### PR TITLE
Extra projects, change code styling strategy

### PR DESCRIPTION
**Pull request by lpil**
### Checklist

* [x] I have read the [Contributor Guide](../../CONTRIBUTING.md)
* [x] I have read and agree to the [Code of Conduct](../../CODE_OF_CONDUCT.md)
* [x] I have added a description of my changes and why I'd like them included in the section below

### Description of Changes

Add missing projects, hiding those we don't want advertised. With a "hidden" property.

####  Linting

I've gone out on a limb and swaped the deprecated eslint action for prettier, which is closer to Rustfmt which we use on Rust projects. This has the nice bonus of not needing errors to be manually fixed, prettier will automatically do it for us.

This doesn't align with the current JS guidelines ([a single line saying we use airbnb style eslint](https://github.com/EmbarkStudios/opensource-template/blob/main/CONTRIBUTING.md#javascript)) but I think it could be worth changing that as prettier has a lot less maintenance overhead than eslint, and the migration of our other JS project can be done by running `npx prettier --write **/*.js`.

What do you think?

Closes: #48

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -148,6 +148,26 @@
...
+    },
+    {
+      "name": "review-action",
+      "emoji": "🏷️",
+      "tags": ["github-actions", "tools"],
+      "description": "Github Action that adds and removes labels based on PR review state"
+    },
+    {
+      "name": "nfd2",
+      "hidden": "This is a temporary fork so we are not showing it on the website"
```
Comment (arirawr) [inline data.json:160]: Clever idea - please add a line to the readme about how `hidden` should be used!

**Comment by arirawr** [comment]
I'm fine with moving to prettier, this PR should also delete `eslintrc` too :)

File: data.json (hunk 1 above)
Comment (lpil) [reply data.json:160]: I was just about to write up docs on all the other properties, and there they are in the README! I am a dummy

**Comment by lpil** [comment]
> this PR should also delete eslintrc too :)

More stuff to delete, sweet

**Comment by arirawr** [comment]
@lpil you can also make the relevant change to the guidelines, saying we use prettier with single quote and whatever other rules.

**Comment by lpil** [comment]
For sure, I'll update the other bits and bobs.

I added the single quote bit to make the diff smaller, but it might be easier to just use the Prettier defaults and tolerate one larger diff. Easier installation and less config.

**Review by arirawr — APPROVED** [review]

**Comment by arirawr** [comment]
Approving this and we can take the JS formatting discussion when we update the guidelines!

**Comment by lpil** [comment]
I'll put that config then :)

**Comment by lpil** [comment]
Huh, turns out the other project already uses Prettier

**Comment by lpil** [comment]
As does review-action

**Comment by arirawr** [comment]
well I guess that gives our answer then! I will live with double-quotes :P

**Pull request merged**